Firestore/core/src/firebase/firestore/remote/grpc_connection.mm

```diff
@@ -118,6 +118,21 @@
                                        observer, worker_queue_);
 }
 
+std::unique_ptr<GrpcStreamingReader> GrpcConnection::CreateStreamingReader(
+    absl::string_view rpc_name,
+    const Token &token,
+    const grpc::ByteBuffer &message) {
+  LOG_DEBUG("Creating gRPC streaming reader");
+
+  EnsureActiveStub();
+
+  auto context = CreateContext(token);
```

Comment:
The types are `std::unique_ptr<grpc::ClientContext>` and `std::unique_ptr<grpc::GenericClientAsyncReaderWriter>`. My reasoning was that the former is largely redundant (because "context" is already in the variable name and the function name) and the latter is unwieldy. I don't feel strongly about this at all, though; if you prefer, I'll inline these types.

Reply:
I don't feel strongly either. I flagged this because it seemed odd to have no type information at all, but I'm fine with your explanation.